tools/pull-request-labeler/src/interfaces/commit-message.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license
+ * Copyright 2019 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export type CommitType =
+  | 'feat'
+  | 'fix'
+  | 'docs'
+  | 'style'
+  | 'refactor'
+  | 'perf'
```

Comment:
whats about commit type ci?

can you use the types in the `./commitlint.config.js`

Reply:
I don't think I can use them, because everything runs in a docker container afterwards. I can copy the types into the container during buildtime, but then I would not be able to run the script locally. 
I will add the ci type though. To be honest, these types are from the old cherry picker, and back then we did not have the `ci` type

Reply:
no probs just saw that this is missing maybe add a comment that those two should be in sync then! I'm fine with just copying was only a recommendation

Reply:
Even though I would create a const enum in case you are using the plain strings at other locations.